dom.bs

```diff
@@ -8810,6 +8832,7 @@ method, when invoked, must run these steps:
   <ol>
    <li><p>If <var>force</var> is passed and is false, return false.
 
+   <li><p>Otherwise, run <a>validation steps</a> with <var>token</var>. If the return value is false, return false.
```

Comment:
This needs to be followed by a newline. Also, here and elsewhere text is not wrapped at 100 columns.